### DISCUSSION TITLE
Awesome Project

### DISCUSSION THREAD

**Discussion by saivan** [prosekit/prosekit#798] [General]
Just wanted to say that I love this project and really love what you're doing here. It's awesome to see you making a crack at this. I love the way you've composed the code here, I hope this project expands heaps soon. Great job!

**Comment by ocavue** [comment]
Thank you for your kind words. I would love to hear feedback from users. Let me know if you have any feature requests or suggestions for improvements, and I will prioritize them.

**Reply by saivan** [reply]
As I'm working with it, I'll file some issues :)

**Comment by MathiasWP** [comment]
I just wanted to swing by and say the same thing! This library is amazing ❤️ And for me as a Svelte developer you’re a lifesaver! By far best Rich/Block text editor for the Svelte ecosystem i’ve found (and i’ve been looking ALOT)!
